pkg/util/file.go

```diff
@@ -37,3 +45,18 @@ func ReplaceFile(destPath string, sourcePath string, perm fs.FileMode) error {
 
 	return nil
 }
+
+// FileGroupID returns gid of the file or directory.
+func FileGroupID(path string) (gid uint32, err error) {
```

Comment:
Since this is a public function, it would be nice to add some tests for this function in `file_test.go`.

Reply:
Will add unit tests in the next revision, thanks for reviewing!